pygeoapi/flask_app.py

```diff
@@ -166,3 +166,7 @@ def serve(ctx, debug=False):
 #    setup_logger(CONFIG['logging'])
     APP.run(debug=True, host=api_.config['server']['bind']['host'],
             port=api_.config['server']['bind']['port'])
+
+
+if __name__ == '__main__':  # run locally, for testing
```

Comment:
Do we need this?  `pygeoapi serve` accomplishes this already.

Reply:
Main reason for `main()` here is for (stepwise) debugging within an IDE like IntelliJ IDEA or WebStorm and probably PyCharm: needed easy entry point.